magento2/magento-base/Dockerfile

```diff
@@ -1,45 +1,12 @@
-FROM php:7.2-apache
+FROM nosto/php-apache:7.3
```

Comment:
Why the Nosto ones? Just curious.

Reply:
Magento uses multiple libraries and extensions that are trimmed from the original images. Such as:
sockets, intl, opcache, mbstring etc.